container/build.sh

```diff
@@ -23,9 +23,10 @@ Usage: build.sh [options]
 Builds the fully-loaded container, with Google Cloud Container Builder or locally.
 
 Required parameters (when build with Google Cloud Container Builder):
+    -d|--distro             Distro of the base image, debian 8, debian 9
```

Comment:
add full list of supported base images here with the same names as expected below (debian8, debian9, ubuntu16_04)